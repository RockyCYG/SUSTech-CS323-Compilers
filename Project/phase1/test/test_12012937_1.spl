struct node{
	int zls
	;
	char 
	cyg;
};
struct node FluffyFluffy[998244353
]
;
int main(int _fff
){

	char kkk='0x#&';
	char ccc
	='888'
	int m=0x233;
	int a = 1;
	int int_ = Fluffy(a);
	
	
	printf(strlen(s=1.2.3....4545));
	zlstxdy>(((233<6)==(luss
	!=
	99
	
	
	
	)));
	Fluffy(fluffy233difjdl839483djf);
	(fluffy[100]=1000!=999)=000;
	111!=1!=2!=555;
}
