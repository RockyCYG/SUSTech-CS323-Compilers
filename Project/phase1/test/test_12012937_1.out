Error type A at Line 13: unknown lexeme '0x#&'
Error type A at Line 15: unknown lexeme '888'
Error type B at Line 14: syntax error, probably Missing SEMI
Error type A at Line 21: unknown lexeme 1.2.3....4545
Error type A at Line 30: unknown lexeme 000
